src/spoke/Spoke.sol

```diff
@@ -200,6 +200,13 @@ contract Spoke is Auth, Recoverable, ReentrancyProtection, ISpoke, ISpokeGateway
         emit AddShareClass(poolId, scId, shareToken_);
     }
 
+    /// @inheritdoc ISpokeGatewayHandler
+    function updateRequestManager(PoolId poolId, ShareClassId scId, AssetId assetId, address manager) public auth {
```

Comment:
We should likely also add `manager` to `NotifyPool` message, to set up on initialization

Reply:
We can batch both messages then, right?

There are cases like subsidizing the hub through a local spoke where we do not need a manager